packages/create-sitecore-jss/src/initializers/nextjs/index.ts

```diff
@@ -54,6 +54,11 @@ export default class NextjsInitializer implements Initializer {
             name: 'nextjs-sxa - Includes example components and setup for Headless SXA projects',
             value: 'nextjs-sxa',
           },
+          {
+            name:
+              'nextjs-sxa-project-playtravel - Includes example components and assets for Headless SXA projects',
```

Comment:
```suggestion
              'nextjs-sxa-project-playtravel - Includes example components and assets for Play Travel project',
```
@sc-dawidrutkowski Is this the correct description?

Reply:
Let's go with this for now. SXA team can change if needed with their PRs.